pkg/resources/prometheusRules.go

```diff
@@ -68,10 +65,6 @@ func (r *AlertReconcilerImpl) ReconcileAlerts(ctx context.Context, client k8scli
 		}
 	}
 
-	if err := r.deleteAlerts(ctx, client, r.RemovedAlerts); err != nil {
-		return integreatlyv1alpha1.PhaseFailed, err
-	}
-
```

Comment:
I'd be more in favor of keeping this code and the relevant tests as this provides a way to remove alerts in the future if alerts are renamed / required to be removed :thinking:

Reply:
sure I could leave it in for the future rather then having to write it again later.